Tests/Parser/Client/fixtures/library.yml

```diff
@@ -659,3 +659,15 @@
     type: library
     name: Boto3
     version: 1.17.18
+-
+  user_agent: Custom-AsyncHttpClient
```

Comment:
Is this (and `Custom HTTP Client`) really a library? I am unable to find anything specific, feels more like a generic name, which should imply "unknown" user agent.